public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -29,9 +31,6 @@ include ../../../../_includes/_util-fns
   * [Template expression operators](#expression-operators)
 // #enddocregion intro
 
-:marked
-  [Run the live example](/resources/live-examples/template-syntax/ts/plnkr.html)
```

Comment:
Notice that I've removed the "Live example" here because.... Which live example are we even talking about? I find this rather misplaced.